src/EditorFeatures/Test/EditAndContinue/ActiveStatementDescription.cs

```diff
@@ -112,7 +112,8 @@ internal static int[] GetIds(string ids)
                 var ids = GetIds(match.Groups["Id"].Value);
                 int absoluteOffset = offset + markedSyntax.Index;
 
-                yield return ValueTuple.Create(new TextSpan(absoluteOffset, markedSyntax.Length), ids);
+                var span = markedSyntax.Length != 0 ? new TextSpan(absoluteOffset, markedSyntax.Length) : new TextSpan();
```

Comment:
When a namespace is deleted the active span is reported as [0..0) however the expected active span is [6..6) (`string src2 = @"<AS:0></AS:0>"`).
The issue only occurs because the test source code includes active statement tags not present in C# code.
It is possible to add a length check to the active statement itself instead but this would not be used in non test code.